demo/core_10_base.js

```diff
@@ -55,14 +55,40 @@ $.utils.commandMenu = function(commands) {
   return cmdXml;
 };
 
+$.utils.jsonStringifyPhysicals = function(value) {
```

Comment:
I think it would be much preferable to do the $.physical -> .name conversion as a separate pass from stringification:
* Less chance of introducing error in stringification.
* Easy to swap in replacer-based implementation later.